### ISSUE TITLE
timestamp array

### ISSUE THREAD

**Issue by AlexandderGorodetski** [k2-fsa/icefall#1419]
Hello guys,

I am looking on the following lines  in greedy_search function. I see that you gather timestamp for every emitting symbol. But these time after subsampling. My question is what equation should I used in order to get real time stamps from the subsampled timestamp vector.

        y = logits.argmax().item()
        if y not in (blank_id, unk_id):
            hyp.append(y)
            timestamp.append(t)
            decoder_input = torch.tensor([hyp[-context_size:]], device=device).reshape(
                1, context_size
            )

Thank you so much,
AlexG.

**Comment by csukuangfj** [comment]
Roughly speaking, you need to scale it by the actual subsampling factor, which is usually 4 in icefall.
